application/templates/static_site/measure.html

```diff
@@ -510,10 +510,10 @@ <h3 class="govuk-heading-s govuk-!-margin-bottom-0">Note on corrections or updat
                 {% if measure_version.secondary_data_source.frequency_of_release %}
                   <h3 class="govuk-heading-s govuk-!-margin-bottom-0">Publication frequency</h3>
                   {% if measure_version.secondary_data_source.frequency_of_release.description == 'Other' %}
-                        {{ measure_version.secondary_data_source.frequency_of_release_other }}
+                    <p class="govuk-body">{{ measure_version.secondary_data_source.frequency_of_release_other }}</p>
                   {% else %}
-                      {{ measure_version.secondary_data_source.frequency_of_release.description }}
-                  {% endif %}
+                    <p class="govuk-body">{{ measure_version.secondary_data_source.frequency_of_release.description }}
+                  {% endif %}</p>
```

Comment:
This needs to be inside the `{% endif %}`, I think.

Reply:
🤦‍♂